.github/workflows/ci.yml

```
@@ -33,7 +33,7 @@ jobs:
     - name: Install dependencies
       run: |
         pip install behave pytest tabulate pyparsing sqlalchemy numpy pydantic pydot sqlalchemy_utils django python-dotenv deprecated pandas pyspellchecker rtree
-        wget -O /tmp/ifcopenshell_python.zip https://s3.amazonaws.com/ifcopenshell-builds/ifcopenshell-python-`python3 -c 'import sys;print("".join(map(str, sys.version_info[0:2])))'`-v0.8.1-0267e2b-linux64.zip
+        wget -O /tmp/ifcopenshell_python.zip https://s3.amazonaws.com/ifcopenshell-builds/ifcopenshell-python-`python3 -c 'import sys;print("".join(map(str, sys.version_info[0:2])))'`-v0.8.1-eadf35f-linux64.zip
```

```suggestion
        wget -O /tmp/ifcopenshell_python.zip https://s3.amazonaws.com/ifcopenshell-builds/ifcopenshell-python-`python3 -c 'import sys;print("".join(map(str, sys.version_info[0:2])))'`-v0.8.1-1d27161-linux64.zip
```
Updated ifcopenshell to this version on the [DEV](https://github.com/buildingSMART/ifc-gherkin-rules/commit/7572e156ecff5b992e7406e83d77bc7039b19803) branch. We should probably also update it here to avoid overwriting.
This version seem also to reflect well your and Rick's work on ifcopenshell

---

We actually need something newer that was just brewed today.  See 3f65633f8d7be02f272d103959227c7070c0fd5d.